pkg/arvo/app/aqua.hoon

```diff
@@ -485,11 +485,10 @@
     =/  =desk  +.val
     =.  userspace-ova.pil
       ::  take all files from a userspace desk
-      =/  all-dirs=(list path)  ~[/]
       :_  ~
       %-  unix-event:pill-lib
-      %+  %*(. file-ovum:pill-lib directories all-dirs)
-      desk  /(scot %p our.hid)/[desk]/(scot %da now.hid)
+      %-  file-ovum:pill-lib  :+  desk
```

Comment:
Using a `:+` here isn't very idiomatic.  It should be either `%^` or something like this:
```hoon
%-  file-ovum:pill-lib
[desk /(scot %p our.hid)/[desk]/(scot %da now.hid) ~[/sys]]
```
I think I'd prefer this over the `%^` since the entire argument can be expressed on a single line.

Reply:
Is excluding /sys correct? I _think_ this change actually represents a slight improvement over the prior implementation, but I have never used `%aqua` so I'm not 100% sure.